tests/unit/test_auth.py

```
@@ -1749,6 +1813,147 @@ def test_loads_cookies_from_all_regional_patterns(self, tmp_path):
+        "youtube.com",
+        "accounts.youtube.com",
+        ".accounts.youtube.com",
+        "drive.google.com",
+        ".drive.google.com",
+        "docs.google.com",
+        ".docs.google.com",
+        "myaccount.google.com",
+        ".myaccount.google.com",
+        "mail.google.com",
+        ".mail.google.com",
+    ]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)